### ISSUE TITLE
Accordian items are not rendering unique id value.

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#1541]
I tried creating an anchor to an accordian item and it is not working. Upon further inspection the id is not rendering at all.

**Comment by briand44** [comment]
Modei is the customer who was having issues with this in case we roll out a fix, we should communicate that to him.

**Comment by GaryRidgway** [comment]
this should be solved in https://github.com/uiowa/uiowa/pull/1548, waiting on pr approval and then merging when we can.
